usaspending_api/api_contracts/contracts/v2/agency/toptier_code/obligations_by_type.md

```diff
@@ -0,0 +1,58 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Federal Obligations [/api/v2/agency/{toptier_code}/obligations_by_type/{?fiscal_year}]
+
+This endpoint is used to power USAspending.gov's agency profile pages.
+
+## GET
+
+This endpoint returns a breakdown of obligations by award type (contracts, IDVs, grants, loans, direct payments, other) within a fiscal year.
+
++ Parameters
+    + `toptier_code`: 086 (required, number)
```

Comment:
This is not a number field. The leading 0 is necessary and is only handled properly as a string

Reply:
String values in the contract should be quoted by backticks, `086`

Reply:
@tony-sappe TY